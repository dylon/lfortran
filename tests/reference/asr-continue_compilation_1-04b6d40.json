{
    "basename": "asr-continue_compilation_1-04b6d40",
    "cmd": "lfortran --semantics-only --continue-compilation --no-color {infile}",
    "infile": "tests/errors/continue_compilation_1.f90",
    "infile_hash": "d97c435b7e36e4222113dcd71cc8549c2f4cc3a3389313356a13c587",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-continue_compilation_1-04b6d40.stderr",
    "stderr_hash": "84786eb0574443e4e9a7a36f25e0c8cbef6532d67403a02c22b2a018",
    "returncode": 1
}