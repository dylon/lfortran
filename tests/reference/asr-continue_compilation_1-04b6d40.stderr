syntax error: Token 'print' is unexpected here
  --> tests/errors/continue_compilation_1.f90:75:5
   |
75 |     print*, nint(1e12_8)
   |     ^^^^^ 

semantic error: Implicit typing is not allowed, enable it by using --implicit-typing 
 --> tests/errors/continue_compilation_1.f90:2:5 - 3:1
  |
2 |        implicit integer(a-f), real(e-z)
  |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...
...
  |
3 |    
  | ...^ 

semantic error: arguments of `repeat` intrinsic must be scalar
  --> tests/errors/continue_compilation_1.f90:86:38
   |
86 |     character(3), parameter :: ar1 = repeat(["abc", "#^1", "123"], [1, 2, 3])
   |                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Assignment to loop variable `i` is not allowed
  --> tests/errors/continue_compilation_1.f90:20:8
   |
20 |        i = i + 1
   |        ^ 

semantic error:  first argument of `maskl` must be less than or equal to the BIT_SIZE of INTEGER(KIND=4)
  --> tests/errors/continue_compilation_1.f90:24:13
   |
24 |     print*, maskl(63)
   |             ^^^^^^^^^ 

semantic error: first argument of `maskr` must be less than or equal to the BIT_SIZE of INTEGER(KIND=4)
  --> tests/errors/continue_compilation_1.f90:26:13
   |
26 |     print*, maskr(63)
   |             ^^^^^^^^^ 

semantic error: first argument of `maskl` must be nonnegative
  --> tests/errors/continue_compilation_1.f90:28:13
   |
28 |     print*, maskl(-24)
   |             ^^^^^^^^^^ 

semantic error: first argument of `maskr` must be nonnegative
  --> tests/errors/continue_compilation_1.f90:30:13
   |
30 |     print*, maskr(-24)
   |             ^^^^^^^^^^ 

semantic error: The argument `matrix_a` in `matmul` must be of type Integer, Real, Complex or Logical
  --> tests/errors/continue_compilation_1.f90:32:21
   |
32 |     print *, matmul(a1, b1)
   |                     ^^ 

semantic error: The argument `matrix_b` in `matmul` must be of type Integer, Real or Complex if first matrix is of numeric type
  --> tests/errors/continue_compilation_1.f90:34:25
   |
34 |     print *, matmul(b1, a1)
   |                         ^^ 

semantic error: The `matmul` intrinsic doesn't handle logical type yet
  --> tests/errors/continue_compilation_1.f90:36:14
   |
36 |     print *, matmul(a2, b1)
   |              ^^^^^^^^^^^^^^ 

semantic error: `matmul` accepts arrays of rank 1 or 2 only, provided an array with rank, 3
  --> tests/errors/continue_compilation_1.f90:38:21
   |
38 |     print *, matmul(a3, b1)
   |                     ^^ 

semantic error: `matmul` accepts arrays of rank 1 or 2 only, provided an array with rank, 4
  --> tests/errors/continue_compilation_1.f90:40:25
   |
40 |     print *, matmul(b1, b4)
   |                         ^^ 

semantic error: The argument `matrix_b` in `matmul` must be of rank 2, provided an array with rank, 1
  --> tests/errors/continue_compilation_1.f90:42:24
   |
42 |     print *, matmul(a, b)
   |                        ^ 

semantic error: `transpose` accepts arrays of rank 2 only, provided an array with rank, 1
  --> tests/errors/continue_compilation_1.f90:44:24
   |
44 |     print *, transpose(a)
   |                        ^ 

semantic error: Kind of all the arguments of Mergebits must be the same
  --> tests/errors/continue_compilation_1.f90:46:14
   |
46 |     print *, merge_bits(8, 12_8, 2)
   |              ^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Kind of all the arguments of Mergebits must be the same
  --> tests/errors/continue_compilation_1.f90:48:14
   |
48 |     print *, merge_bits(a5, b5, c5)
   |              ^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Second argument of modulo cannot be 0
  --> tests/errors/continue_compilation_1.f90:64:14
   |
64 |     print *, modulo(1, 0)
   |              ^^^^^^^^^^^^ 

semantic error: Function 'my_func' not found (not user defined nor intrinsic)
  --> tests/errors/continue_compilation_1.f90:66:5
   |
66 |     call my_func(y=1, x=2, z=1)
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Result of `nint` overflows its kind(4)
  --> tests/errors/continue_compilation_1.f90:76:13
   |
76 |     print*, nint(1000000000000.0000000000000000d0)
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Invalid argument `hello` supplied
  --> tests/errors/continue_compilation_1.f90:78:5
   |
78 |     OPEN(file="numbers", hello="world")
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Cannot assign to a constant variable
  --> tests/errors/continue_compilation_1.f90:80:5
   |
80 |     i1 = 3
   |     ^^^^^^ assignment here
   |
10 |     integer, parameter :: i1 = 2
   |                           ~~~~~~ declared as constant

semantic error: Expected 0 or 1 arguments, got 2 arguments instead.
  --> tests/errors/continue_compilation_1.f90:82:5
   |
82 |     call FLUSH(1, 2)
   |     ^^^^^^^^^^^^^^^^ 

semantic error: `kind` argument of `verify` intrinsic must be a scalar
  --> tests/errors/continue_compilation_1.f90:84:39
   |
84 |     print*, verify(string, set, kind= [4, 4] )
   |                                       ^^^^^^ 

semantic error: arguments of `and` intrinsic must be scalar
  --> tests/errors/continue_compilation_1.f90:85:14
   |
85 |     print *, and([1, 2, 3], [1, 2, 3])
   |              ^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: The shift argument of 'dshiftl' intrinsic must be less than or equal to the bit size of the integer
  --> tests/errors/continue_compilation_1.f90:88:28
   |
88 |     print *, dshiftl(1, 2, 34)
   |                            ^^ 

semantic error: The shift argument of 'dshiftl' intrinsic must be non-negative integer
  --> tests/errors/continue_compilation_1.f90:89:28
   |
89 |     print *, dshiftl(1, 2, -2)
   |                            ^^ 

semantic error: The shift argument of 'dshiftr' intrinsic must be less than or equal to the bit size of the integer
  --> tests/errors/continue_compilation_1.f90:91:28
   |
91 |     print *, dshiftr(1, 2, 34)
   |                            ^^ 

semantic error: The shift argument of 'dshiftr' intrinsic must be non-negative integer
  --> tests/errors/continue_compilation_1.f90:92:28
   |
92 |     print *, dshiftr(1, 2, -2)
   |                            ^^ 

semantic error: arguments of `selected_int_kind` intrinsic must be scalar
  --> tests/errors/continue_compilation_1.f90:94:14
   |
94 |     print *, selected_int_kind([1,2,3])
   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: arguments of `selected_real_kind` intrinsic must be scalar
  --> tests/errors/continue_compilation_1.f90:95:14
   |
95 |     print *, selected_real_kind([1,2,3])
   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: arguments of `selected_char_kind` intrinsic must be scalar
  --> tests/errors/continue_compilation_1.f90:96:14
   |
96 |     print *, selected_char_kind(['c', 'a', 'b'])
   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
